FirebaseAuth/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 10.16.0
 - [added] Added custom auth domain support in recaptcha v2 authentication flows. (#7553)
+- [added] Added host override support for IdentityToolKit and SecureToken api endpoints. (#11858)
```

Comment:
This will be at least 10.17.0 since 10.16.0 is already frozen.

Please move to an `# Unreleased` section.